tests/utilities/test_all_gather_grad.py

```diff
@@ -67,7 +66,7 @@ def training_epoch_end(self, outputs) -> None:
                 "losses": losses,
                 "losses_list": [losses, losses]
             })
-            assert gathered_loss["losses_tensor_int"][0].dtype == torch.int64
+            assert gathered_loss["losses_tensor_int"][0].dtype == torch.int32
```

Comment:
so I assume this test didn't run, and you had to fix this here :)

Reply:
Yes. The test was silently failing before